common/protob/options.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto2";
+package hw.trezor.messages;
+
+// Sugar for easier handling in Java
+option java_package = "com.satoshilabs.trezor.lib.protobuf";
+option java_outer_classname = "TrezorOptions";
+
+import "google/protobuf/descriptor.proto";
+
+/************************* WARNING ***********************
+Due to the way extensions are accessed in pb2py, there needs to be a globally unique
+name-ID mapping for extensions. That means that two different extensions, e.g. for
+EnumValueOptions and FieldOptions, MUST NOT have the same ID.
+
+Using the same ID indicates the same purpose (protobuf does not allow multiple
+extensions with the same name), such as EnumValueOptions.bitcoin_only and
+FileOptions.include_in_bitcoin_only. pb2py can then find the extension under
+either name.
+
+The convention to achieve this is as follows:
+ - extensions specific to a type have the same prefix:
+   * 50xxx for EnumValueOptions
+   * 51xxx for EnumOptions
+   * 52xxx for MessageOptions
+   * 53xxx for FieldOptions
+ - extensions that might be used across types have the same "global" prefix 60xxx
+*/
+
+/**
+ * Options for specifying message direction and type of wire (normal/debug)
+ */
+ extend google.protobuf.EnumValueOptions {
+    optional bool wire_in = 50002;              // message can be transmitted via wire from PC to Trezor
+    optional bool wire_out = 50003;             // message can be transmitted via wire from Trezor to PC
+    optional bool wire_debug_in = 50004;        // message can be transmitted via debug wire from PC to Trezor
+    optional bool wire_debug_out = 50005;       // message can be transmitted via debug wire from Trezor to PC
+    optional bool wire_tiny = 50006;            // message is handled by Trezor when the USB stack is in tiny mode
+    optional bool wire_bootloader = 50007;      // message is only handled by Trezor Bootloader
+    optional bool wire_no_fsm = 50008;          // message is not handled by Trezor unless the USB stack is in tiny mode
+    optional bool channel_in = 50009;
+    optional bool channel_out = 50010;
+    optional bool pairing_in = 50011;
+    optional bool pairing_out = 50012;
+
```

Comment:
Please add these in a separate commit so that it's clear this one is just a refactor (bonus points for using `refactor(common):` in the commit message). Also please add docstrings.